databases/db.js

```diff
@@ -22,7 +23,9 @@ const Promotion = PromotionsModel(sequelize, Sequelize); //Nueva implementacion
 const game = gamesModel(sequelize, Sequelize);
 const team = teamsModel(sequelize, Sequelize);
 const inventory = inventoryModel(sequelize, Sequelize);
-const Deposit = 
+//const Deposit =
+const PlayerFantasy= playerFantasyModel(sequelize,Sequelize);
+//PlayerFantasy = playerFantasyModel
```

Comment:
```suggestion
const PlayerFantasy= playerFantasyModel(sequelize,Sequelize);
```
Si no lus usan, mejor quitarlos que dejarlos comentados xd

Reply:
Sii, se me olvidó quitar eso :(